packages/office-addin-telemetry/README.md

```diff
@@ -1,8 +1,8 @@
-# Office-Addin-telemetry
-This package allows for sending telemetry event and exception data to the selected telemetry infrastructure (e.g. ApplicationInsights).
+# Office-Addin-Usage-Data
+This package allows for sending usage-data event and exception data to the selected usage-data infrastructure (e.g. ApplicationInsights).
```

Comment:
"usage-data" here should be "usage data". To make the sentence more understandable, change to something more succinct e.g. "This package allows for reporting usage data events."